bundle.proto

```diff
@@ -109,3 +109,102 @@ message BundleResult {
     Dropped dropped = 6;
   }
 }
+
+/** BundleStatus types */
+
+message FinalizedOnChain {
```

Comment:
does it make sense to combine processed and finalized and use a commitment level?

Reply:
also what about confirmed? i think most searchers will use processed and confirmed